content/en/docs/k8s-operator/automatic.md

```diff
@@ -108,7 +108,65 @@ is required for auto-instrumentation to work properly.
 
 ### .NET
 
-Coming Soon
+The following command will create a basic Instrumentation resource that is
+configured specifically for instrumenting .NET services.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: demo-instrumentation
+spec:
+  exporter:
+    endpoint: http://demo-collector:4318
+  propagators:
+    - tracecontext
+    - baggage
+  sampler:
+    type: parentbased_traceidratio
+    argument: "1"
+EOF
+```
+
+By default, the Instrumentation resource that auto-instruments .Net services
+uses `otlp` with the `http/proto` protocol. This means that the configured
+endpoint must be able to receive OTLP over `http/proto`. Therefore, the example
+uses `http://demo-collector:4318`, which will connect to the `http` port of the
+otlpreceiver of the Collector created in the previous step.
+
+By default, the .NET auto-instrumentation ships with
+[many instrumentation libraries](/docs/instrumentation/net/libraries/#available-instrumentation-libraries).
```

Comment:
This is a lot for "SDK based" instrumentation libraries.
The libraries supported by auto-instrumentation can be found here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md#instrumentations

Reply:
Changed to link to the repo config doc.